holmes/core/conversations_worker/worker.py

```
@@ -272,8 +272,9 @@ def _realtime_verify_loop(self) -> None:
             try:
                 result = self.dal.is_realtime_enabled()
             except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)